tests/baselines/reference/argumentsSpreadRestIterables(target=es5).types

```diff
@@ -24,10 +24,10 @@ declare const itNum: Iterable<number>
 
 ;(function(a, ...rest) {})('', true, ...itNum)
 >(function(a, ...rest) {})('', true, ...itNum) : void
->(function(a, ...rest) {}) : (a: string, rest_0: boolean, ...rest_1: any[]) => void
->function(a, ...rest) {} : (a: string, rest_0: boolean, ...rest_1: any[]) => void
+>(function(a, ...rest) {}) : (a: string, rest_0: boolean, ...rest_1: Iterable<number>[]) => void

```

Comment:
This test targets `es5` where `Iterable` doesn't exist. This change shows how now an unresolved type continues to be displayed with its alias name~. Just like here:
```ts
type Test = {
  foo: Thing; // even though it's unresolved and errors, it is still displayed as `Thing`
};
```

And actually, this matches the display of those IIFEs above this one. So it turns out that this one was going through `createNormalizedTupleType` that accidentally lost this display value.